routes/user.playlist.js

```diff
@@ -252,139 +255,210 @@ router.route('/:playlist_name/export3')
             .exec(function(err, playlists) {
                 if (err) return next(err)
                 songs2d = playlists.map((playlist) => {
-                        return {
-                            songs: playlist.translationsChecked
-                                // minLine: _.min(playlist.translationsChecked.map((t) => t.lyric.length))
-                        }
+                        return playlist.translationsChecked
+                            // minLine: _.min(playlist.translationsChecked.map((t) => t.lyric.length))
+
                     })
                     // songs2d = playlists.map((playlist) => playlist.translationsChecked)
                 next()
             })
     })
     .get(function(req, res, next) {
-        var type = req.query.type;
+        if (type == 1) {
```

Comment:
as I mentioned, instead of using "type" which is overused in programming, call it languagesPerSlide.